src/Attributes/MapFrom.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Spatie\DataTransferObject\Attributes;
+
+use Attribute;
+use Spatie\DataTransferObject\Caster;
+use Spatie\DataTransferObject\Exceptions\InvalidCasterClass;
+
+#[Attribute(Attribute::TARGET_CLASS | Attribute::TARGET_PROPERTY)]
+class MapFrom
+{
+    public function __construct(
+        protected string | int $attribute,
```

Comment:
Make this one public, so that you can use the attribute's instance (see the other comment)

Reply:
Fixed in 6fd88a379b34965ea3c9ea687b84e4fc09a213c9.